### ISSUE TITLE
Health Professional ResearchKit / CareKit configuration

### ISSUE THREAD

**Issue by md0u80c9** [ResearchKit/ResearchKit#646]
Hi,

Just something I thought might be worth mulling over, particularly with the CareKit announcement. Apologies that I'm providing the use case rather than a solution / implementation idea at this stage - hoping it gives someone an idea.

ResearchKit applications currently are very reliant upon self-setup: in other words independent, technologically-competent 'expert' patients who configure the system themselves.

But, if we think about the 'process' of managing chronic diseases - these interactions start with a health professional, who we want to then 'hand over' to a model of self-care or research to the patient, rather than just relying on patient self-initiation.

Clearly expert patient self-configuration only suits a small subset of patients - either clinical knowledge may need entering, or configuration may be better done by a health professional. But those professionals lack time to do this entirely manually, and their time is expensive and valuable. CareKit takes this need a step further where transfer of already known clinical details is really important - and there may be a lot of such data or it may be complex to encapsulate with manual data entry (a resting ECG for example).

In terms of my day-to-day usage - I'm a stroke physician. A small number of my patients could set up a stroke app - but the vast majority either can't or won't; the barriers are many and simply too big (in both the 'can't' and the 'won't' group).

We would get much greater usage or value from such applications if I could send every iPhone-owning patient home (or their next of kin or main carer) with a copy of their key results, a summary of their stroke admission, and appropriate stroke prevention advice on their device. That might even include a copy of their scans if they wished for example. Much of our research involves looking at outcomes at up to 6 months post-stroke, so we could also configure the information we want back from them over the next 6 months and set up appropriate ResearchKit apps for them (haemorrhagic strokes being recruited to different trials to ischaemic strokes for example).

Whilst our clinical team is mostly fairly iPhone-savvy, and we actually use a cohort of iPad minis on the ward, I can't imagine we can reasonably stretch this to a team spending their time logging onto the App Store, downloading apps for strangers, configuring them and despatching them with the device pre-installed; and the time involved would be huge given the turnover of our department. We have up to 10 discharges a day: if configuration of iPhones took them 20-30 minutes each we would need to recruit a full-time person just to do that: clearly that's not going to be viable!

Instead we need the secure physical transfer of data between a trusted physician or researcher's device, and a trusted patient's device. The patient may not have the application on their phone, so this process must begin with the installing and configuration of the application, but may then need to proceed with the transfer of salient data to configure the application with.

It also needs to be consistent - so if the patient is involved in three research trials, or a piece of research and some self-management with CareKit. If they go to a different department, who uses ResearchKit apps from third-parties, you would want the same configuration steps to be usable. That way - you can publicise your mobile strategy to your patients in a simple and easy way and/or train people to follow a couple of steps process. Think along the lines of the usability of Apple Pay for the user, with client-side a 'select patient and push' model.

As an example, my ResearchKit interest relates around collating data for the Stroke Sentinel National Audit - a UK registry of stroke patients. I could potentially do a lot with both ResearchKit and CareKit with patients. It is important that we transfer their SSNAP ID number to them (a 6 digit unique identifier); but if we want them to self-manage their BP I may need to transfer all their existing BP recordings to them too. If I'm studying cognition, being able to transfer their baseline cognitive test results would clearly be invaluable. I also need to transfer the dates that certain assessments should be done by - so that the application would know what to prompt for and when.

This configuration shouldn't be iOS-specific - we may need to think that the professional software we are transferring data from may not be OS X / iOS-based (because many aren't); although clearly making it easy to write an iOS app which manages a group of client patients running a particular CareKit/ResearchKit app would be hugely valuable.

Hope that's interesting and/or useful!

Andrew

**Comment by pthealthtech** [comment]
You for taking the time to write up this post, Andrew. A question about how to get the app on the patient's phone, is that something that can be done during registration? That work has to be done by someone somewhere. Perhaps the case manager/Care Coordinator?

Kind regards,
John

**Comment by pthealthtech** [comment]
"Thank" you.

**Comment by md0u80c9** [comment]
Yes the commonest opportunities to set up a phone would be at admission (if a patient is stable - think routine outpatients) or at discharge (if they were not - eg. In my stroke patient cohort).

However, that has to involve as little patient-side configuration as possible - you might want them to download a single app and pair up. Or if there was consideration at system level some sort of ApplePay-type connection using Bluetooth where proximity to an appropriate source with a simple-to-operate 2 factor authentication kicked in before transferring data.

**Comment by evliu** [comment]
@md0u80c9 , I think the best way to go about it is to have the medical information recorded into an EHR/EMR system, then expose an API for a ResearchKit app to log into so the patient can retrieve that information, then the app itself would present actionable items depending on how the app is developed.

The pattern of ResearchKit apps at this point is that their data tends to be totally anonymous when sent to the researchers through different organizations and back-end services. What you describe could be done, but it would still be a custom-developed app, which doesn't necessarily involve ResearchKit or CareKit itself, since those are more of toolboxes to provide tools for collecting data and handling things like HIPAA.

In effect, your app would basically need to be downloaded on the patient device, then they need to log in, and from there, can retrieve all their information just as fast as bluetooth from a server that already has all the records. If the app isn't taking any more data, then could the data be sent from the server straight to the researchers?

**Comment by smravca-apple** [comment]
Hey @md0u80c9 wanted to check in on this and see if there are any updates or suggestions of how we can help.
